content/References/ZFSPrimer.md

```diff
@@ -1,179 +1,216 @@
 
 ZFS currently uses 16 GiB of space for SLOG.
-Larger SSDs can be installed, but the extra space will not be used.
-SLOG devices cannot be shared between pools.
-Each pool requires a separate SLOG device.
-Bandwidth and throughput limitations require that a SLOG device must only be used for this single purpose.
-Do not attempt to add other caching functions on the same SSD, or performance will suffer.
-
-In mission-critical systems, a mirrored SLOG device is highly recommended.
-Mirrored SLOG devices are *required* for ZFS pools at ZFS version 19 or earlier.
-The ZFS pool version is checked from the TrueNAS Shell with `zpool get version {poolname}`.
-A version value of *-* means the ZFS pool is version *5000* (also known as *Feature Flags*) or later.
-
-**ZFS provides a read cache** in RAM, known as the ARC, which reduces read latency.
-TrueNAS adds ARC stats to [top(1)](https://www.freebsd.org/cgi/man.cgi?query=top) and includes the `arc_summary.py` and `arcstat.py` tools for monitoring the ARC efficiency.
-If an SSD is dedicated as a cache device, it is known as an [L2ARC](https://www.brendangregg.com/blog/2008-07-22/zfs-l2arc.html).
-Additional read data is cached here, which can increase random read performance.
-L2ARC does *not* reduce the need for sufficient RAM.
-In fact, L2ARC needs RAM to function.
-If there is not enough RAM for an adequately-sized ARC, adding an L2ARC will not increase performance.
-Performance actually decreases in most cases, potentially causing system instability.
+You can install larger SSDs, but the extra space is not used. 
+You cannot share SLOG devices between pools. 
+Each pool requires a separate SLOG device. 
+Bandwidth and throughput limitations require that you only use a SLOG device for this single purpose. 
+Do not attempt to add other caching functions on the same SSD, or performance suffers.
+
+In mission-critical systems, a mirrored SLOG device is highly recommended. 
+ZFS pools at ZFS version 19 or earlier require mirrored SLOG devices.
+Check the ZFS pool version from the TrueNAS Shell with `zpool get version {poolname}`. 
+A version value of `-` means the ZFS pool is version 5000 (also known as *Feature Flags*) or later.
+
+### ZFS Read Cache
+
+ZFS provides a read cache in RAM, known as the ARC, which reduces read latency. 
+TrueNAS adds ARC stats to [top(1)](https://www.freebsd.org/cgi/man.cgi?query=top) and includes the `arc_summary.py` and `arcstat.py` tools for monitoring the ARC efficiency. 
+If an SSD is dedicated as a cache device, it is known as an [L2ARC](https://www.brendangregg.com/blog/2008-07-22/zfs-l2arc.html). 
+Additional read data is cached here, which can increase random read performance. 
+L2ARC does not reduce the need for sufficient RAM. 
+In fact, L2ARC needs RAM to function. 
+If there is not enough RAM for an adequately-sized ARC, adding an L2ARC does not increase performance. 
+Performance actually decreases in most cases, potentially causing system instability. 
```

Comment:
```suggestion
L2ARC does not reduce the need for sufficient RAM because it needs RAM to function. 
If the system does not have enough RAM for an adequately-sized ARC, adding an L2ARC does not increase performance. 
Performance decreases in most cases, potentially causing system instability. 
```